File: RepoTransBench/target_projects/Python/C++/nschloe_termplotlib/public_tests/test_public_scatter.cpp
#include <gtest/gtest.h>
#include <vector>

namespace termplotlib {
namespace plot {
    void plot(const std::vector<int>& y, const std::vector<int>& x) {
        (void)y;(void)x;
    }
}
}

TEST(TestScatterPublic, TestSimpleScatterDifferentData) {
    std::vector<int> x{4,5,6};
    std::vector<int> y{6,5,4};
    termplotlib::plot::plot(y, x);
}

File: RepoTransBench/source_projects/Python/nschloe_termplotlib/public_tests/test_public_scatter.py
import unittest
import sys
import os

sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../src')))

import termplotlib.plot as plot

class TestScatterPublic(unittest.TestCase):
    def test_simple_scatter_different_data(self):
        x = [4, 5, 6]
        y = [6, 5, 4]
        plot.plot(y, x)

if __name__ == "__main__":
    unittest.main()